cpp/tests/test_flows.cpp

```diff
@@ -119,9 +117,12 @@ TEST(TestFlows, FlowSimulation)
     model.parameters.set<mio::oseir::TransmissionProbabilityOnContact<double>>(0.04);
     model.parameters.get<mio::oseir::ContactPatterns<double>>().get_cont_freq_mat()[0].get_baseline().setConstant(10);
 
+    mio::set_log_level(mio::LogLevel::off);
```

Comment:
Perhaps one can put a comment here why it was taken off and then set back to warn.

Reply:
This silences the simulation (or rather the OdeIntegrator). I will add a short comment.